src/store/modules/pagination/reducers.ts

```diff
@@ -0,0 +1,25 @@
+import { FLATS, FLATS_MAX } from "./types";
+import { paginationAction, initialPaginationState } from "./types";
+
+const paginationReducers = (state = initialPaginationState, action: paginationAction) => {
+    let payload;

```

Comment:
czy ta zmienna jest tu niezbedna? Bo z tego co widze to mozesz uzyc bezposrednio action.payload